crates/compiler/builtins/bitcode/src/panic.zig

```diff
@@ -0,0 +1,16 @@
+const std = @import("std");
+const RocStr = @import("str.zig").RocStr;
+const always_inline = std.builtin.CallOptions.Modifier.always_inline;
+
+// Signals to the host that the program has panicked
+extern fn roc_panic(msg: *const RocStr, tag_id: u32) callconv(.C) void;
+
+pub fn panic_help(msg: []const u8, tag_id: u32) void {
+    var str = RocStr.init(msg.ptr, msg.len);
+    roc_panic(&str, tag_id);
+}
+
+// must export this explicitly because right now it is not used from zig code
```

Comment:
I know this comment was already here but I don't think it's actually true, so maybe worth checking while we're editing stuff!

Reply:
It's still needed because we need to `@export` it